src/store/store.ts

```diff
@@ -1,13 +1,17 @@
-import * as Y from 'yjs'
-import { TDBinding, TDShape } from '@tldraw/tldraw'
-import { WebsocketProvider } from 'y-websocket';
+import * as Y from "yjs";
+import { TDBinding, TDShape } from "@tldraw/tldraw";
+import { WebsocketProvider } from "y-websocket";
 
-export const doc = new Y.Doc()
+export const doc = new Y.Doc();
 export const urlParams = new URLSearchParams(window.location.search);
-export const roomID = urlParams.get('roomName') ?? 'GLOBAL';
-export const provider = new WebsocketProvider(`ws://localhost:3345`, roomID, doc, {
-  })
+export const roomID = urlParams.get("roomName") ?? "GLOBAL";
+export const provider = new WebsocketProvider(
+  `ws://localhost:3345`,
```

Comment:
Please make it configurable, you'll probably need to use a similar mechanism for the configuration as the one that the _nuxt-client_ is using.